### PR TITLE
feat: masked s2s secret keys

### PR DESCRIPTION
**Pull request by KOTLAPRA23**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-1775

### Change description ###

 masked s2s secret keys

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/functionalTest/resources/application-functional.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ idam.auth.clientId: rd-professional-api
 
 idam.auth.tokenAuth: ${IDAM_TOKEN_ATUH:cmQtcHJvZmVzc2lvbmFsLWFwaTpjYzVmMmE2LTk2OTAtMTFlOS1iYzQyLTUyNmFmNzc2NGY2NA==}
 idam.auth.redirectUrl: ${OAUTH2_REDIRECT_URI:https://rd-professional-api-aat.service.core-compute-aat.internal/oauth2redirect}
```
Comment (shreedhar-hmcts) [inline src/functionalTest/resources/application-functional.yaml:15]: I think oauth2.secret and IDAM_TOKEN_ATUH can also be removed as it is unused, please check.

File: src/integrationTest/resources/application.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ spring:
         registration:
           oidc:
             client-id: microservicekey-rd-professional-api
-            client-secret: ${S2S_SECRET:K6KA6PRLRFWQLSD6}
+            client-secret: ${S2S_SECRET:AAAAAAAAAAAAAAAA}
```
Comment (shreedhar-hmcts) [inline src/integrationTest/resources/application.yml:25]: We can also remove client.secret which is used for sidam bearer token

Comment (KOTLAPRA23) [reply src/integrationTest/resources/application.yml:25]: where it is  used ?

Comment (KOTLAPRA23) [reply src/integrationTest/resources/application.yml:25]: it is masked  now to some value

File: src/functionalTest/resources/application-functional.yaml (hunk 1 above)
Comment (KOTLAPRA23) [reply src/functionalTest/resources/application-functional.yaml:15]: i am not doing code cleanup now...later we can do

**Pull request closed without merging**